### PR TITLE
initial n6stomp commit. PEP8 ready!

### PR DESCRIPTION
**Pull request by aaronkaplan**
Please review. It's supposed to be PEP8 ready.
Also I had some initial reviews with @sebix .

@robcza : maybe you can check it as well and see if it works for you :)

### CODE REVIEW

**Comment by aaronkaplan** [comment]
yes, I know I am still missing some tests. But apart from that - we should be OK.

File: intelmq/bots/collectors/n6/collector_stomp.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    """ the stomp listener gets called asynchronously for
+        every STOMP message
+    """
+    def __init__(self, n6stompcollector):
+        self.n6stomper = n6stompcollector
+
+    def on_heartbeat_timeout(self):
+        # XXX FIXME: use logger instead of print
+        # XXX FIXME: need to reconnect after timeout
+        print "lost connection!"
```
Comment (sebix) [inline intelmq/bots/collectors/n6/collector_stomp.py]: As already said, a no-go!

Comment (aaronkaplan) [inline intelmq/bots/collectors/n6/collector_stomp.py]: nice catch, I forgot that one. Okay, fixed. Apart from that minor print thing: did you test it??

Comment (sebix) [inline intelmq/bots/collectors/n6/collector_stomp.py]: Please be patient, I will be in the office in 12 hours...

File: intelmq/bots/BOTS [hunk 2]
```diff
@@ -460,6 +460,20 @@
                 "rate_limit": 3600,
                 "http_url": "http://vxvault.siri-urz.net/URL_List.php"
             }
+        },
+        "N6stomp": {
```
Comment (sebix) [inline intelmq/bots/BOTS]: Not in sorted order.

**Pull request merged**